chrome/locale/en-US/zotero/zotero.dtd

```diff
@@ -127,7 +127,8 @@
 
 <!ENTITY zotero.toolbar.feeds.new					"New Feed…">
 <!ENTITY zotero.toolbar.feeds.new.fromURL			"From URL…">
-<!ENTITY zotero.toolbar.feeds.new.fromPage		"From Page…">
+<!ENTITY zotero.toolbar.feeds.new.fromPage			"From Page…">
+<!ENTITY zotero.toolbar.feeds.new.fromOPML			"Import From OPML…">
```

Comment:
Let's go with "From OPML…" for consistency with the other menu items. ("From OPML File…" would be a bit more precise, but probably not necessary.)